src/robot/reporting/jsmodelbuilders.py

```diff
@@ -22,9 +22,10 @@
 class JsModelBuilder(object):
 
     def __init__(self, log_path=None, split_log=False,
-                 prune_input_to_save_memory=False):
+                 prune_input_to_save_memory=False, auto_expand_keywords=[]):
```

Comment:
Mutable default values are always a bit questionable. Would be better to use None or possibly () here.

Reply:
refactoring to None. It's used in JsBuildingContext, which now also supports None.

Reply:
refactoring to None + implemented handling of None